src/helpers/parser/sueParser.js

```diff
@@ -28,21 +28,21 @@ export const parseSueData = (data, appDesignSystem) => {
 
     return {
       ...item,
-      address: item.address.replace('\r\n ', '\r\n'),
+      address: item.address?.replace('\r\n ', '\r\n'),
       aspectRatio: parsedImageAspectRatio('361:203'),
       bottomDivider: false,
-      iconName: matchedStatus.iconName,
+      iconName: matchedStatus?.iconName,
       params: {
-        title: item.title,
+        title: item?.title,
         query: QUERY_TYPES.SUE.REQUESTS_WITH_SERVICE_REQUEST_ID,
-        queryVariables: { id: item.serviceRequestId },
+        queryVariables: { id: item?.serviceRequestId },
```

Comment:
an `item` should be always present here, we do not need to check it

Reply:
I will also remove the controls in the code here.
```
const mediaArray = item.mediaUrl || JSON.parse(item.media_url);
```

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/966/commits/2b603bd8f6f8b43bf449efd90fce208da3b2c23c